src/js/utils/log.js

```diff
@@ -24,7 +26,7 @@ export const logByType = (type, args, stringify = !!IE_VERSION && IE_VERSION < 1
   // Was setting these once outside of this function, but containing them
   // in the function makes it easier to test cases where console doesn't exist
   // when the module is executed.
-  const fn = console && console[type] || function(){};
+  const fn = console && console[type] || function() {}; // eslint-disable-line
```

Comment:
The `eslint-disable-line` is there because of the reference to `console`. We _could_ use an arrow function, but I'm trying to avoid any changes that aren't necessary.